pages/previewModel.js

```diff
@@ -120,13 +124,14 @@ function updatePreview() {
     // give them all the same special ID so that we can easily remove them later.
     // We then do something similar for link (CSS) elements.
     clearRemovableHeadElements();
-    var scriptElements = content.getElementsByTagName('script')
-    for (var i = 0; i < scriptElements.length; i++) {
+    var i;
```

Comment:
So in JavaScript, `var` is function-scoped, not block-scoped.  As such, jsHint was warning me that `i` had been declared twice in the same function, and the re-declaration was not technically permitted.  ES6 fixes this by introducing `let` which is a block-scoped declaration.  Probably I could fix this by telling jsHint to treat the code as ES6, and change the `var`s to `let`s.